src/sql/workbench/browser/modelComponents/text.component.ts

```diff
@@ -140,6 +141,6 @@ export default class TextComponent extends TitledComponent<azdata.TextComponentP
 	}
 
 	public get showDiv(): boolean {
-		return this.requiredIndicator || !!this.description;
+		return !!this.description;
```

Comment:
Why is this changing?

Reply:
currently, a div is being created around the text, requiredIndicator, and description. I thought that a div is not necessary when adding the requiredIndicator.

with this logic change, the div will only show up when a description is added to the text. @Charles-Gagnon What are your thoughts?

```
<p> 
  <span>text</span> 
  <span>*</span> 
</p>

<div> 
  <span>text</span>
  <span>*</span> 
  <div>description</div> 
</div>